frontend/app-preview/src/views/LandingPage.module.css

```
@@ -15,11 +19,17 @@
   border: none;
 }
 
-.iframeMobileViewContainer {
+.previewLimitationsInfo {
+  align-self: center;
+  width: 80%;
```

Skulle vi heller satt en max-width istedenfor å bruke prosent? Da vil det skallere bedre når parent-container blir mindre. Dersom parent container bli mindre enn eks. 480px, så vil den ta hele bredden. Hva tenker du? 🤔 

```suggestion
   max-width: 480px;
   width: 100%;
```